.github/workflows/release-send-mattermost-announcement.yml

```diff
@@ -30,9 +34,4 @@ jobs:
       run: |
         echo "{\"text\":\":che-logo: Che ${{ github.event.inputs.version }} release final checks have failed:\n\n ${{join(steps.check.outputs.*, '\n')}}.\"}" > mattermost.json
     - name: Send message
-      if: ${{ success() }} || ${{ failure() }}
-      uses: mattermost/action-mattermost-notify@1.1.0
-      env:
-        MATTERMOST_WEBHOOK_URL: ${{ secrets.MATTERMOST_WEBHOOK_URL }}
-        MATTERMOST_CHANNEL: eclipse-che-releases
-        MATTERMOST_USERNAME: che-bot
+      run: echo > mattermost.json
```

Comment:
this will send a single empty line to mattermost.json. Is that what you intended?